### ISSUE TITLE
Tasks can not be deleted

### ISSUE THREAD

**Issue by olpeh** [olpeh/wht#74]
Make it possible to somehow delete tasks from a project

**Comment by threepistons** [comment]
Workaround, assuming that you have `sqlite3` installed in your SailfishOS device. This is not pleasant, even with a bluetooth keyboard or SSH session.
1. Make sure that the tasks you plan to delete have no events attached to them. I do not know what will happen if you delete a task that has events.
2. Use `sqlite3` on the hexstring.sqlite file in /home/nemo/.local/share/harbour-workinghourstracker. `.headers on` and then `select * from tasks ;` will get you the task table contents, and then you can delete the unwanted rows.

**Comment by olpeh** [comment]
Yup that's nice that you can do it directly on the db level, but not very normal user friendly :sweat_smile:
